doc/API.md

```diff
@@ -86,3 +86,36 @@ This command does not take any parameter for now.
 | `outpoint`     | string        | Transaction id and output index of this coin                        |
 | `block_height` | int or null   | Blockheight the transaction was confirmed at, or `null`             |
 | `block_time`   | int or null   | Timestamp of the block containing the output transaction, or `null` |
+
+### `gethistory`
+
+`gethistory` retrieves a paginated list of accounting events.
+
+Aiming at giving an accounting point of view, the amounts returned by this call are the total
+of inflows and outflows net of any change amount (that is technically a transaction output, but not a cash outflow).
+
+#### Request
+
+| Field         | Type         | Description                                                          |
+| ------------- | ------------ | -------------------------------------------------------------------- |
+| `start`       | int          | Timestamp of the beginning of the period to retrieve events for      |
+| `end`         | int          | Timestamp of the end of the period to retrieve events for            |
+| `limit`       | int          | Maximum number of events to retrieve                                 |
+
+#### Response
+
+| Field          | Type   | Description                                |
+| -------------- | ------ | ------------------------------------------ |
+| `events`       | array  | Array of [Event resource](#event-resource) |
+
+##### Event Resource
+
+| Field         | Type          | Description                                                                                                             |
+| ------------- | ------------- | ----------------------------------------------------------------------------------------------------------------------- |
+| `blockheight` | int           | Blockheight of the event final transaction                                                                              |
+| `txid`        | string        | Hex string  of the event final transaction id                                                                           |
+| `kind`        | string        | Type of the event. Can be `receive`, `spend`                                                                            |
+| `date`        | int           | Timestamp of the event                                                                                                  |
+| `amount`      | int or `null` | Absolute amount in satoshis that is entering or exiting the wallet, `null` if the event is a `cancel` event             |
+| `miner_fee`   | int or `null` | Total of the miner fees caused by the operation, `null` if the event is a `deposit` event                               |
+| `coins`       | string array  | List of outpoints of coins affected by the event excluding any change coin                                              |
```

Comment:
So.. It's essentially the list of spent outpoints for a `spend` event? Is it worth having it for deposits?